[invalid.py]
  [test_params_subscriptions_invalid_type[True\]]
    expected: ERROR

  [test_params_subscriptions_invalid_type[42\]]
    expected: ERROR

  [test_params_subscriptions_invalid_type[subscriptions3\]]
    expected: ERROR

  [test_params_subscriptions_invalid_type[foo\]]
    expected: ERROR

  [test_params_subscriptions_entry_invalid_type[None\]]
    expected: ERROR

  [test_params_subscriptions_entry_invalid_type[True\]]
    expected: ERROR

  [test_params_subscriptions_entry_invalid_type[42\]]
    expected: ERROR

  [test_params_subscriptions_entry_invalid_type[subscription3\]]
    expected: ERROR

  [test_params_subscriptions_entry_invalid_type[subscription4\]]
    expected: ERROR

  [test_params_subscriptions_invalid_value[subscriptions0\]]
    expected: ERROR

  [test_params_subscriptions_invalid_value[subscriptions1\]]
    expected: ERROR

  [test_unsubscribe_with_subscription_id_twice]
    expected: ERROR
